pkg/kubelet/cm/dra/resources.go

```diff
@@ -45,73 +45,73 @@ type resource struct {
 	// podUIDs is a set of pod UIDs that reference a resource
 	podUIDs sets.Set[string]
```

Comment:
Why is this not showing up as part of the diff? This has definitely changed from what I reviewed before.

Reply:
It was included into the [previous PR](https://github.com/pohly/kubernetes/pull/40/commits/9ae288e605d36adaba3d34005856ee6d89825ecc#diff-ba116f7e162cc14649dcf4f8f619379da85fbe15db383b097a69de115e6f601cL46). Some of the linters I run also suggested to use generic Set.